### PR TITLE
8170389: java.text.DigitList.getDouble() : Controversy between javadoc and code

### PR DESCRIPTION
**Pull request by justin-curtis-lu**
Problem: Outdated doc does not match code. Claimed to throw exception and compared to Long method.
Fix: Update doc to match code, compared to Double.parseDouble() accordingly.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8170389](https://bugs.openjdk.org/browse/JDK-8170389): java.text.DigitList.getDouble() : Controversy between javadoc and code

### Reviewers
 * [Naoto Sato](https://openjdk.org/census#naoto) (@naotoj - **Reviewer**) ⚠️ Review applies to [55679168](https://git.openjdk.org/jdk/pull/10567/files/55679168b90bb41f7a2a669c62f429073d8f650f)
 * [Lance Andersen](https://openjdk.org/census#lancea) (@LanceAndersen - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/10567/head:pull/10567` \
`$ git checkout pull/10567`

Update a local copy of the PR: \
`$ git checkout pull/10567` \
`$ git pull https://git.openjdk.org/jdk pull/10567/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10567`

View PR using the GUI difftool: \
`$ git pr show -t 10567`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/10567.diff">https://git.openjdk.org/jdk/pull/10567.diff</a>

</details>

### CODE REVIEW

**Review by naotoj — APPROVED** [review]
Looks good to me. Since you are only modifying the comments, please add `noreg-doc` label to the JIRA issue.

File: src/java.base/share/classes/java/text/DigitList.java [hunk 1]
```diff
@@ -194,6 +194,9 @@ public final long getLong() {
         return Long.parseLong(temp.toString());
     }
 
+    /**
+     * Utility routine to get the value of the digit list.
```
Comment (LanceAndersen) [inline src/java.base/share/classes/java/text/DigitList.java:198]: new BigDecimal("") will also throw a NumberFormatException so I would add a similar comment for count as done above for getLong/Double()

Comment (justin-curtis-lu) [reply src/java.base/share/classes/java/text/DigitList.java:198]: Thanks Lance, will make that change

**Comment by jddarcy** [comment]
Sorry; didn't look closely enough before applying the csr label.

**Comment by justin-curtis-lu** [comment]
> Looks fine Justin.
> 
> Please make sure the comments are < 80 characters and if they are longer break them into two lines as they seem like they might be long

@LanceAndersen Made the change, thanks Lance

**Review by LanceAndersen — APPROVED** [review]
thank you Justin :-)

**Pull request closed without merging**